File: 기초-브론즈 문제/백준_3052_나머지/나머지.java
import java.util.*;

public class Main{
    public static void main(String[] args){
        Scanner s = new Scanner(System.in);
        int[] arr = new int[42];

        int num = 10;
        int a = 0;
        while(num-- > 0){
            a = s.nextInt();
            arr[a%42]++;
        }
        int cnt = 0;
        for(int i = 0; i< arr.length; i++){
            if(arr[i] > 0) cnt++;
        }
        System.out.println(cnt);
    }
}

File: 기초-브론즈 문제/백준_3052_나머지/나머지.kt
import java.util.*

fun main(args: Array<String>) {
    val s = Scanner(System.`in`)
    var arr = IntArray(42, {0})
    var num = 10
    var a = 0

    while(num-- > 0){
        a = s.nextInt()
        arr[a%42] += 1
    }
    var cnt = 0
    for(i in 0..41){
        if(arr[i] > 0) cnt+=1
    }
    print(cnt)

}